docs/releases/2.11.3.rst

```diff
@@ -0,0 +1,43 @@
+============================
+Wagtail 2.11.3 release notes
+============================
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+What's new
+==========
+
+Bug fixes
+~~~~~~~~~
+
+ *
+
+
+Upgrade considerations
+======================
+
+If you have used the Wagtail project template, or have migrations that create page instances programmatically, you will
+need to add ``run_before = [('wagtailcore', '0053_locale_model')]`` to the corresponding ``Migration`` class.
```

Comment:
@zerolab Is it worth expanding on what will happen if you do not? Providing an example error message may help with the discovery of this updgrade tip